integration_tests/test_emigration.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+
+
+def test_individuals_emigrate(simulants_on_adjacent_timesteps):
+    # This is a very rare event, so we can't assert that it happens
+    # on every timestep; instead, we aggregate across all timesteps.
+    all_individual_emigration_status = []
+
+    for before, after in simulants_on_adjacent_timesteps:
+        households_remaining_in_us = after[after["in_united_states"]]["household_id"]
+        individual_emigrants = (
+            before["in_united_states"]
+            & ~after["in_united_states"]
+            & after["household_id"].isin(households_remaining_in_us)
+        )
+
+        # Only occurs to living, tracked people
+        living_tracked = before["tracked"] & (before["alive"] == "alive")
+        assert living_tracked[individual_emigrants].all()
+
+        assert after["time"].nunique() == 1
+        current_time = after["time"].iloc[0]
+        assert (after[individual_emigrants]["exit_time"] == current_time).all()
+
+        all_individual_emigration_status.append(individual_emigrants[living_tracked])
+
+    all_individual_emigration_status = pd.concat(
+        all_individual_emigration_status, ignore_index=True
+    )
+    # VERY conservative upper bound on how often this should be occurring, as a proportion
+    # of living, tracked simulant-steps
+    assert 0 < all_individual_emigration_status.mean() < 0.1
+
+
+def test_non_gq_individuals_emigrate(simulants_on_adjacent_timesteps):
```

Comment:
Lots of this looks like a copy/paste job from the test above. Consider making a helper function that these two tests call so you DRY

Reply:
I have done this in #148.

Reply:
Yeah I just saw that and came here to comment